src/sugar/orderbook.ts

```diff
@@ -9,7 +9,7 @@ import {
   TakerAmount,
 } from '../models/methods/bookOffers'
 
-import { orderFlags } from './parse/flags'
+const SELL_OFFER_FLAGS = 0x00020000
```

Comment:
Is it not possible to get this number from one of the `Flags` things that we already have? Seems bad to have it as a separate constant.

Reply:
Right, so these flags are `lsf` flags. The flags we have defined are `tf` flags. 

So,  we can define these flags in the `./src/models/ledger` directory with the associated ledger entry.

Reply:
I'd just add a TODO for now then - not worth doing in this PR

Reply:
Just added it.